Bender.yml

```diff
@@ -21,8 +21,8 @@ sources:
   # Files in level 1 only depend on files in level 0, files in level 2 on files in levels 1 and 0,
   # etc. Files within a level are ordered alphabetically.
   # Level 0
-  - src/cdc_2phase.sv
   - src/cf_math_pkg.sv
+  - src/cdc_2phase.sv
```

Comment:
Is there a reason why you moved `cdc_2phase`? We generally keep an alphabetical order amongst dependencies.